workflows/Discord_Message_Notify.yml

```diff
@@ -0,0 +1,30 @@
+name: Discord Message Notify
```

Comment:
Since we talked about another notify action, maybe we should rename this one to `Discord submission notify`

Reply:
Yep good call, I'll try convert all mention of it on the github over too.